doc/tutorial/staramr-tutorial.ipynb

```diff
@@ -54,35 +54,35 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "--2019-04-10 15:17:07--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/478/105/GCF_001478105.1_Salmonella_enterica_CVM_N31384-SQ_v1.0/GCF_001478105.1_Salmonella_enterica_CVM_N31384-SQ_v1.0_genomic.fna.gz\n",
+      "--2019-05-07 12:17:37--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/478/105/GCF_001478105.1_Salmonella_enterica_CVM_N31384-SQ_v1.0/GCF_001478105.1_Salmonella_enterica_CVM_N31384-SQ_v1.0_genomic.fna.gz\n",
```

Comment:
Thanks so much for taking the time to update this file :smile: 

It looks like for the `plasmidfinder.tsv` and `summary_detailed.tsv` files the older **Gene** column names are still displayed:

![image](https://user-images.githubusercontent.com/200517/57326989-5bad3880-70d3-11e9-91fc-6c83d89e00dc.png)

Reply:
That's strange, thought I already updated when I first ran through it, maybe not. Changed it in 22b711b

Reply:
Thanks :smile:. Those changes all look good.

Note that there's a few places where the text **plasmid genes** appears. Could you change them to **plasmid type**?

Reply:
Sure will do, fixed that in 061fdf8